management-commands/config.md

```diff
@@ -111,7 +123,7 @@ moderator. It will change the role to a temporary role and remove it after a cer
 
 MoonlightBot requires the following permissions to successfully execute this subcommand:
 
-*No specific permissions required*
+*View Audit Log*
```

Comment:
Revert, does not go here. The actual command does not require any permission, it's the role assignment detection itself that needs the audit log.

Reply:
So the bot in general needs the permission to view the audit log?

Reply:
yes, it must be a separate annotation, better if warning hint that it's needed

Reply:
Okay, I started with the hint, but it felt wrong